packages/components/src/components/drawer/readme.md

```diff
@@ -0,0 +1,126 @@
+# Drawer (Sidebar) `preview`
+
+<kol-alert _type="warning" _variant="card">
+  <kol-badge _color="#476af5" _label="Preview"></kol-badge> Diese neue Komponente wird als ungetestet markiert, da die Barrierefreiheitstests noch ausstehen. Die verschiedenen Tests können aufgrund der Modularität bei neuen Komponenten und Funktionalitäten meist erst nach einem Release erfolgen. Wir empfehlen daher, die Komponente noch nicht in Produktion zu verwenden.
+</kol-alert>
+
+Mit Hilfe der **Drawer**-Komponente können zusätzliche Informationen oder auch Navigationselemente in einem ausklappbaren Seitenfenster angezeigt werden. Ein offener Drawer kann als Modal (\_modal) via ESC geschlossen werden. Die **Drawer**-Komponente ist standardmäßig versteckt. Sie wird i.d.R. erst nach Klick auf einen Button oder sonstigem Trigger angezeigt bzw. controlled mit dem Attribut \_open gesteuert.
+
+```html
+<kol-drawer _label="Drawer" _open _modal _align="top" _on={{ onClose: () => void }}> </kol-drawer>
```

Comment:
Das `_on`-Attribut funktioniert in der HTML-bzw. Webcomponent-Variante so leider nicht. Würde ich für das Beispiel wohl einfach weglassen.